### ISSUE TITLE
Settings Disclosure Arrow

### ISSUE THREAD

**Issue by mpermann** [grokability/snipe-it#11129]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

When I click the Settings disclosure arrow to expose the options below and then click one of those options the disclosure arrow collapses and hides the options. This doesn't happen when I do the same operations for the Reports disclosure arrow. Is there a setting I'm missing or is this a bug? Searching outstanding issues didn't find other reports and I've seen the same behavior in the latest versions of Safari, Chrome and Firefox on macOS Monterey 12.4.

### Reproduction steps

1. Log into SnipeIT console. 
2. Click the sidebar button to expose the sidebar if it's collapsed.
3. Click the Settings disclosure arrow to expose the options below Settings.
4. Click on one of the options in the exposed list of settings to select it and observe that the disclosure arrow collapses.

### Expected behavior

I would expect that once you've exposed the options under Settings using the disclosure arrow, they would stay exposed like happens for the Reports disclosure arrow. 

### Snipe-IT Version

6.0.1

### Operating System

Ubuntu 20.04.4

### Web Server

Apache

### PHP Version

7.4.3

### Operating System

macOS Monterey 12.4

### Browser

Chrome, Firefox and Safari

### Version

101.0.4951.64, 100.0.1, 15.5

**Comment by snipe** [comment]
That's the way it has always behaved. We can consider adding a cookie to keep the disclosed submenus open, I suppose, though it's likely not everyone would want that.

**Comment by mpermann** [comment]
It seems strange that one (Settings) of the three items (Assets, Settings and Reports) with disclosure arrows behaves differently. It would be great if you could add the ability for the Settings disclosure arrow to behave the same as the Assets and the Reports disclosure arrow. I suppose it's possible that some folks may not like this behavior, so it might be worth adding an option in "Branding" to disable it individually or as a group. Thanks for responding!

**Comment by snipe** [comment]
Interesting, I hadn't noticed that the Settings behaved differently. I'm not sure why that would be - they all use kind of the same basic code. @Godmartinz can you take a look please?

**Comment by mpermann** [comment]
No problem. I just started playing around with your product and noticed this and was frustrated by it so I thought I would mention it. I'm really impressed with all the improvements you've made to the product. I played with it years ago and it was missing some things that we had in our custom built system. But now it looks like you have everything we would need. Whether I can convince them to consider it as an option is something all together different. Thanks for being so responsive to issues!
